js/fitbit-service.js

```diff
@@ -0,0 +1,185 @@
+(function() {
+    'use strict';
+
+    var express = require('express');
+    var app     = express();
+    var fs      = require( 'fs' );
+    var Fitbit  = require( 'fitbit-oauth2' );
+
+    function FitbitService($http) {
+        var service = {};
+        var summary = null;
+        var today = null;
+        
+        // Simple token persist functions.
+        //
+        var tfile = 'fb-token.json';
```

Comment:
The initial access token is a once off thing - never needs to be done again :smile:  

The refresh token is then called whenever the access token expires, and the fitbit-oauth2 library that is included handles refreshing this with a refresh token automagically. No fuss or extra work required by the user.

Basically, they do the setup as detailed in my fitbit readme file I included, and then don't ever have to touch this bit again. Whenever an API call is made to the fitbit API, if the access token is invalid, the library automatically requests a new refresh token using the Client ID and secret in the fitbit-service.js config.

This is from the official Fitbit API docs:

"Your application stores the access token and refresh token. It will use the access token to make requests to the Fitbit API. It will use the refresh token to obtain a new access token when the access token expires without having to re-prompt the user."

I like your idea on a remote mobile configuration app though, but for this Fitbit integration its something that is done once off, and not needed to be done again. Once that feature in #48 is done, I would be happy to look at moving the Fitbit initial configuration to that app to make it easy for 'normal users'. :smile: